1.9/storage/persistent-volume.md

```diff
@@ -73,10 +71,10 @@ The second volume is a persistent volume with a `containerPath` that matches the
 
 For a complete example, see the [Running stateful MySQL on Marathon](#stateful-sql) section.
 
-## Creating a stateful application via the DC/OS Web Interface
+## Creating a stateful application via the DC/OS GUI
```

Comment:
yes, GUI is the official term. there are occurrences of web interface and UI throughout - but our goal is to change to GUI :)